services/worker-manager/schemas/v1/register-worker-response.yml

```diff
@@ -36,7 +36,18 @@ properties:
     required:
       - accessToken
       - clientId
+  workerConfig:
+    type: object
+    title: Worker Config
+    additionalProperties: true
+    description: |
+      This value is supplied unchanged as the `workerConfig` property of the `taskcluster` instance metadata attribute.
+      The expectation is that the worker will merge this information with configuration from other sources,
+      and this is precisely what [taskcluster-worker-runner](https://github.com/taskcluster/taskcluster-worker-runner) does.
+      This property must not be used for secret configuration, as it is visible both in the worker pool configuration and in the worker instance's metadata.
+      Instead, put secret configuration in the [secrets service](https://github.com/taskcluster/taskcluster-worker-runner#secrets).
```

Comment:
I think this needs updated, since in this case it's not in the metadata.  Same for the similar property in `config-static.yml`